src/view/matcher.js

```diff
@@ -130,7 +130,7 @@ export default class Matcher {
 	 *			pattern: <pattern used to match found element>,
 	 *			match: {
 	 *				name: true,
- 	 *				attributes: [ 'title', 'href' ],
+	 *				attribute: [ 'title', 'href' ],
 	 *				classes: [ 'foo' ],
```

Comment:
My bad, I haven't looked at other keys, I had a problem with `attribute(s)` only (and it made me really angry, BTW.)